sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -182,12 +204,13 @@ def _get_metric_value(self, metric):
     else:
       return None
 
-  def _get_metrics_from_dataflow(self):
+  def _get_metrics_from_dataflow(self, job_id=None):
```

Comment:
Being able to query any job's metrics is great : )

Reply:
ack